packages/firebase_dynamic_links/firebase_dynamic_links_platform_interface/lib/src/match_type.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// The match type of the Dynamic Link.
+/// https://firebase.google.com/docs/reference/ios/firebasedynamiclinks/api/reference/Enums/FIRDLMatchType.html
+enum MatchType {
+  ///  The match has not been achieved.
+  none,
+
+  /// The match between the Dynamic Link and this device may not be perfect, hence you should
+  /// not reveal any personal information related to the Dynamic Link.
+  weak,
+
+  /// The match between the Dynamic Link and this device has high confidence but small possibility
+  /// of error still exist.
+  defaultMatchType,
```

Comment:
Not sure I'm a fan in the variations of the naming convention for this one - but I'm also not sure what a better alternative could be when `default` is a keyword in Dart 🤷

Reply:
@Salakar Yeah, exactly. Will change to whatever is best but we can't have `default` unfortunately.